chsdi/templates/htmlpopup/windatlas50.mako

```diff
@@ -0,0 +1,1003 @@
+<%inherit file="base.mako"/>
+<%!
+from chsdi.lib.raster.georaster import get_raster
+
+r = get_raster('COMB')
```

Comment:
I'd rather use the official service here (yes, means roundtrip...). But having it here measn having dtm data on all instances that we have this popup. This is officially not the case on our production instances, where we have this data only for the profile/height service instances (it's separated). And in the future, this service might be completely separated from chsdi3. In that moment, we have to use the service any way.

It's better to decouple this now.